spec/features/search_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+RSpec.feature 'searching for a place' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.